### PR TITLE
Added a restart button to domain server settings page

### PR DESCRIPTION
**Pull request by seefo**
Should fix https://highfidelity.fogbugz.com/f/cases/5367/Add-an-option-to-restart-the-domain-server-from-the-web-interface

Test Plan:
1. Start domain server
2. Visit domain server settings page
3. Click the restart button, which can be found on the navbar
4. Verify that the domain server restarts

### CODE REVIEW

**Comment by CainFoool** [comment]
Will test once built.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/868/HighFidelity-Beta-PR10745-0c7f1cda0f7828b1952f5af40febdf3eb3e80aaa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/868/HighFidelity-Beta-PR10745-0c7f1cda0f7828b1952f5af40febdf3eb3e80aaa.exe

**Comment by CainFoool** [comment]
# Test Passed 

Button appears on the navigation bar at the top, reminds you that the server is restarting in three seconds, the page refreshes, my client temporarily disconnects from my local Sandbox and re-connects momentarily.

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

File: domain-server/resources/web/js/domain-server.js [hunk 1]
```diff
@@ -7,4 +32,10 @@ $(document).ready(function(){
   $('ul.nav a').filter(function() {
       return this.href == url;
   }).parent().addClass('active');  
+
+  $('body').on('click', '#restart-server', function(e){
+    $.get("/restart");
```
Comment (birarda) [inline domain-server/resources/web/js/domain-server.js]: We should show a restart modal if we get a 200, and show an error modal if not

**Pull request merged**